model_request_example.py

```diff
@@ -188,6 +240,75 @@ async def claude_service(request: TextGenerate):
         return {"status_code": 500, "content": {"message": str(e)}}
 
 
+async def llama_service(request: TextGenerate):
```

Comment:
There's a lot of overlap between `claude_service` and `llama_service`. Maybe try to reduce the duplication through helper functions.